packages/applications-service-api/src/services/representation.service.js

```diff
@@ -1,20 +1,72 @@
+const { Op } = require('sequelize');
 const db = require('../models');
 const config = require('../lib/config');
 
-const getRepresentationsForApplication = async (applicationId, page) => {
+const getRepresentationsForApplication = async (applicationId, page, searchTerm, type) => {
   const { itemsPerPage: limit } = config;
   const offset = (page - 1) * limit;
-  const where = { CaseReference: applicationId };
+  let where = { CaseReference: applicationId };
+  if (type) where = { ...where, RepFrom: { [Op.in]: type } };
```

Comment:
We should just keep one form of query object - i.e. the form with the [Op.and] key at the top-level.   Then this line could become:

```
if (type) {
      where[Op.and].push({
        RepFrom: { [Op.in]: type },
      });
    }
```

The code between lines 32 and 36 can then also be removed.